bench/irmin/data/analysis/bench_fixed_size_string_set.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
Shouldn't this script be moved to  `tarides/tezos-storage-bench`?

Reply:
This is a hold-over from not having rebased over https://github.com/mirage/irmin/pull/1770.

Reply:
Oh ok, thanks!